scheduler/src/cook/mesos/api.clj

```diff
@@ -1479,10 +1488,15 @@
   (base-retries-handler
     conn is-authorized-fn
     {:allowed-methods [:put]
-     :exists? (partial check-jobs-and-groups-exist conn)
      :malformed? (partial validate-retries conn task-constraints)
-     :handle-created (partial display-retries conn)
-     :put! (partial retry-jobs! conn)}))
+     :exists? (partial check-jobs-and-groups-exist conn)
+     :put! (partial retry-jobs! conn)
+     ;; :put-enacted? decides whether to respond with Accepted (false) or Created (true).
+     :put-enacted? (comp seq ::jobs)
```

Comment:
https://httpstatuses.com/202 says:

> The request has been accepted for processing, but the processing has not been completed.

It's not really true in this case that processing has not completed, right?

Reply:
Changed to 200 OK.